response_operations_ui/views/passwords.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+from flask import abort, redirect, render_template, request, url_for, Blueprint, current_app as app
+from itsdangerous import URLSafeSerializer, BadSignature, SignatureExpired, BadData
+from structlog import wrap_logger
+
+from response_operations_ui.forms import ForgotPasswordForm, ResetPasswordForm
+from response_operations_ui.controllers import uaa_controller
+from response_operations_ui.controllers.notify_controller import NotifyController
+from response_operations_ui.exceptions.exceptions import NotifyError
+from response_operations_ui.common import token_decoder
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+BAD_AUTH_ERROR = 'Unauthorized user credentials'
+
+passwords_bp = Blueprint('passwords_bp', __name__, static_folder='static', template_folder='templates')
+
+
+@passwords_bp.route('/forgot-password', methods=['GET'])
+def get_forgot_password():
+    form = ForgotPasswordForm(request.form)
+    return render_template('forgot-password.html', form=form)
+
+
+@passwords_bp.route('/forgot-password', methods=['POST'])
+def post_forgot_password():
+    form = ForgotPasswordForm(request.form)
+    form.email_address.data = form.email_address.data.strip()
+    email = form.data.get('email_address')
+
+    if form.validate():
+        return send_password_change_email(email)
+
+    return render_template('forgot-password.html', form=form, email=email)
+
+
+@passwords_bp.route('/forgot-password/check-email', methods=['GET'])
+def forgot_password_check_email():
+    encoded_email = request.args.get('email', None)
+
+    if encoded_email is None:
+        logger.error('No email parameter supplied')
+        return redirect(url_for('passwords_bp.get_forgot_password'))
+
+    try:
+        email = URLSafeSerializer(app.config['SECRET_KEY']).loads(encoded_email)
+    except BadSignature:
+        logger.error('Unable to decode email from URL', encoded_email=encoded_email)
+        abort(404)
+
+    return render_template('forgot-password-check-email.html', email=email)
+
+
+@passwords_bp.route('/reset-password/<token>', methods=['GET'])
+def get_reset_password(token, form_errors=None):
+    form = ResetPasswordForm(request.form)
+
+    try:
+        duration = app.config['EMAIL_TOKEN_EXPIRY']
+        token_decoder.decode_email_token(token, duration)
```

Comment:
Should this be `token = token_decoder.decode_email_token(token, duration)`?  or do we not care about what the token decodes into?
If it's the latter then this should be `_ = token_decoder.decode_email_token(token, duration)` to show that we're purposefully discarding the output